sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/DicomIOReadIT.java

```diff
@@ -60,6 +61,7 @@ public void deleteDicomStore() throws IOException {
     client.deleteDicomStore(healthcareDataset + "/dicomStores/" + storeName);
   }
 
+  @Ignore
```

Comment:
Please create a JIRA to track the issue, and link it here.

Reply:
Can I label it as a "bug"?

Reply:
https://issues.apache.org/jira/browse/BEAM-11376

Reply:
> Can I label it as a "bug"?

Yes. I added it to the "test-failures" component as well.